src/cim/components/helpers/versions.ts

```diff
@@ -2,13 +2,8 @@ import uniqBy from 'lodash/uniqBy';
 import { AgentClusterInstallK8sResource, ClusterImageSetK8sResource } from '../../types';
 import { OpenshiftVersionOptionType, OpenshiftVersion } from '../../../common';
 
-export const getVersionFromReleaseImage = (releaseImage = '') => {
-  const match = /.+:(.*)-/gm.exec(releaseImage);
-  if (match && match[1]) {
-    return match[1];
-  }
-  return '';
-};
+export const getVersionFromReleaseImage = (releaseImage = '') =>
+  (releaseImage.split(':')[1] || '').split('-')[0];
```

Comment:
I think it would be a bit better to take the last part of the `split` ,not the second one
```suggestion
  const releaseImageParts = releaseImage.split(':')
  return releaseImageParts[releaseImageParts.length-1] || '').split('-')[0]
```

Reply:
Updated, thanks.